src/index.spec.ts

```diff
@@ -270,6 +336,201 @@ describe("calculateDepreciation()", () => {
       }
     ]);
   });
+
+  it("should not return endAmount of 3", () => {
```

Comment:
Does this test (and the one below with “endAmount of 5”) test anything specific that isn’t already covered by the tests above? In other words, is it just another “example” with different values or does it actually test a specific case?

For example, I understand having tests for `2016-01-01` and `2016-05-01` (because they need to treated quite differently), however, there’s arguably not much more benefit in testing `2016-06-01` or `2016-07-01`.

Reply:
yes exactly only show the possibility of remaining amount more than 1, like 3 and 5, if in future will be any change in calculation, this is good to be remained this tests to cover that. I agree with you, with this new approach for calculation, this tests may be not much more benefits but may be show the improvement and previous challenges.